accepted/build/build-publish-tweaks.md

```diff
@@ -0,0 +1,28 @@
+# Build and Publish tweaks in the .NET Core SDK
+
+In .NET Core 3, we are making Build output more similar to the Publish output.  In .NET Core 2.x and prior, dependencies from NuGet packages are not copied into the Build output.  At runtime, they are loaded from the NuGet package cache.  This meant that the Build output was not portable to other machines.
```

Comment:
For what it's worth, NuGet folks would really prefer it if the global packages folder was not referred to as a cache in specs and docs :)
There's no expiration policy there, it's an install location.